src/tests/ftest/util/command_utils.py

```diff
@@ -76,7 +77,7 @@ def update(self, value, name=None):
         """
         self.value = value
         if name is not None:
-            print("Updated param {} => {}".format(name, self.value))
+            self.log.info("Updated param %s => %s" % (name, self.value))
```

Comment:
(pylint-logging-not-lazy) Specify string format arguments as logging function parameters